build-challenge/package.json

```diff
@@ -5,24 +5,41 @@
   "description": "Host a build challenge within a survival world",
   "private": true,
   "devDependencies": {
-    "del": "^6.0.0",
-    "gulp": "^4.0.2",
-    "gulp-cli": "^2.3.0",
-    "gulp-sourcemaps": "^3.0.0",
-    "gulp-typescript": "^6.0.0-alpha.1",
-    "gulp-zip": "^5.1.0",
+    "@minecraft/core-build-tasks": "../resources/minecraft-core-build-tasks-1.0.1-patch-30.tgz",
+    "eslint-plugin-minecraft-linting": "^1.0.1",
+    "npm-watch": "^0.11.0",
     "source-map": "^0.7.4",
-    "typescript": "^4.4.3"
+    "ts-node": "^10.9.1",
+    "typescript": "^5.0.2"
   },
   "scripts": {
+    "lint": "just-scripts lint",
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "package": "just-scripts package",
+    "local-deploy": "just-scripts local-deploy",
+    "mcaddon": "just-scripts mcaddon",
+    "updateworld": "just-scripts updateworld",
+    "watch": "npm-watch",
     "enablemcloopback": "CheckNetIsolation.exe LoopbackExempt -a -p=S-1-15-2-1958404141-86561845-1752920682-3514627264-368642714-62675701-733520436",
     "enablemcpreviewloopback": "CheckNetIsolation.exe LoopbackExempt -a -p=S-1-15-2-424268864-5579737-879501358-346833251-474568803-887069379-4040235476"
   },
+  "watch": {
+    "local-deploy": {
+      "patterns": [
+        "scripts",
+        "behavior_packs",
+        "resource_packs"
+      ],
+      "quiet": false,
+      "extensions": "ts,json,lang,png"
+    }
+  },
```

Comment:
hmm, is there a reason why this can't just be handled via a local-deploy parameter to minimize the need to install the `npm-watch` pacakge? That package doesn't seem to have recent updates or movement

Reply:
Same for other cases

Reply:
Still in progress